packages/catalog-client/src/CatalogClient.test.ts

```diff
@@ -193,6 +193,29 @@ describe('CatalogClient', () => {
 
       expect(response.items).toEqual([]);
     });
+
+    it('handles ordering properly', async () => {
+      expect.assertions(2);
+
+      server.use(
+        rest.get(`${mockBaseUrl}/entities`, (req, res, ctx) => {
+          expect(req.url.search).toBe('?order=kind&order=-metadata.name');
```

Comment:
is the minus in the `order=-metadata.name` intended?

Reply:
Yes. It's how you encode "please use inverse lexicographical sort", basically. Inspired by the syntax used in some other query APIs, `+`/`-` meaning `asc`/`desc` respectively. But you can leave out `+`.